### PR TITLE
Lab3 letter frequency

### PR DESCRIPTION
**Pull request by vinilapeddireddy**
Fixes #63 

#### Changes proposed in this PR:
* Implemented letter frequency
* Write a function that returns a sorted slice of strings with elements "{key}:{val}"
* Add unit test

### CODE REVIEW

File: 03_letters/vinilapeddireddy/main.go
Comment (juliaogris) [inline 03_letters/vinilapeddireddy/main.go:17]: Blank line between functions.

File: 03_letters/vinilapeddireddy/main.go
Comment (juliaogris) [inline 03_letters/vinilapeddireddy/main.go]: `"%s:%d"`

File: 03_letters/vinilapeddireddy/main_test.go
Comment (juliaogris) [inline 03_letters/vinilapeddireddy/main_test.go]: Remove blank line.

**Review by juliaogris — CHANGES_REQUESTED** [review]
Very close, looking good!

Rebase onto master.
Update commits not to include any mention of fibonacci and use imperative mood (see [slides](https://gotraining.xyz/gotraining.slide#29) for rebasing help.)

File: 03_letters/vinilapeddireddy/main.go
Comment (vinilapeddireddy) [reply 03_letters/vinilapeddireddy/main.go:17]: Done

File: 03_letters/vinilapeddireddy/main.go
Comment (vinilapeddireddy) [reply 03_letters/vinilapeddireddy/main.go]: Done

File: 03_letters/vinilapeddireddy/main_test.go
Comment (vinilapeddireddy) [reply 03_letters/vinilapeddireddy/main_test.go]: Done

**Review by juliaogris — APPROVED** [review]
LGTM 😻

**Comment by juliaogris** [comment]
Once you have approval you should be able to just click the `Update branch` button and without further reapproval merge.

**Pull request merged**
